src/main.cc

```diff
@@ -177,7 +178,28 @@ void finish_warmup()
     SCHEDULING_LATENCY = 0;
     EXEC_LATENCY = 0;
     DECODE_LATENCY = 2;
+// Runtime PAGE_TABLE_LATENCY
+// if environment var is defined, set to that value
+char* s = getenv("CS_PAGE_TABLE_LATENCY");
+if(!s) {
     PAGE_TABLE_LATENCY = 100;
+	printf("Page Table latency set to default of %d\n", PAGE_TABLE_LATENCY);
+}
+else {
+	sscanf(s, "%d", &PAGE_TABLE_LATENCY);
+	printf("Page Table latency set to %d\n", PAGE_TABLE_LATENCY);
```

Comment:
Here's an example of blending C++ I/O and C-style I/O. You can do exactly the same with `cin`, `cout` and `clog`. More details [here](https://en.cppreference.com/w/cpp/io).

It would basically look like this:
```suggestion
        std::cin >> PAGE_TABLE_LATENCY;
	std::cout << "Page Table latency set to default " << PAGE_TABLE_LATENCY << std::endl;
```

Reply:
Careful! Your line 189 does not keep the same functionality.

```suggestion
	PAGE_TABLE_LATENCY = std::atoi(s);
	std::cout << "Page Table latency set to " << PAGE_TABLE_LATENCY << std::endl;
```

Reply:
Yes, you’re right I read too fast and took it for a scanf.